db/people.yml

```diff
@@ -1396,10 +1396,15 @@ fbertolotti:
   position: assistant professor
 fkwei:
   active: true
-  aka: []
+  aka:
+    - Fengkai Wei
   avatar: https://github.com/Fengkai-Wei.png
   bio: bio
-  education: []
+  education:
+    - begin_year: 2020
+        degree: MS in Material Science and Engineering
+        institution: columbiau
+        end_year: 2021
```

Comment:
end_date not needed because this is your current degree, but your past degrees are missing in this  list.  Please check other examples to get the schema right